prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -4,7 +4,7 @@ elements:
     hl7Spec: [ '%{hl7OBXField}-1' ]
 
   - name: result-value-type-st
-    condition: '%resource.value is String or %resource.value.exists().not()'
+    condition: '%resource.value is String or (%resource.dataAbsentReason.coding.where(extension(%`rsext-code-index-name`).value = "identifier").exists() and %resource.dataAbsentReason.coding.where(extension(%`rsext-code-index-name`).value = "identifier").code = "unknown")'
```

Comment:
I'm not 100% sure if this is actually correct, but this condition is setup to keep the existing CO tests working.  There are several instances where OBX.2 is `ST` in the expected output (and input), but OBX.5 is not valued.

Reply:
I don't think so.  We have a ticket in the backlog that will clean up/remove some of these tests and we can likely update the logic then.  I've been trying to stick with not having to delete/break any of the outputs

Reply:
Do we interpret `C(R/X)` to mean based on the Predicate, it's either required or excluded? Not sure where the key is for the `Usage` column... But anyway, yea that is interesting, the CO tests seem to be pretty arbitrary, they basically just confirm that the behavior matches what it was at a point in time. So I could see us considering changing this to match NIST specs, if that test was the only reason for this, but yea that test ticket might be a better place for that, we should probably note this there though, it'll be easy to lose track of. Will add a brief note on the ticket.